Quantum-Control-Applications/Superconducting/Single Fixed Transmon/Use Case 3 - DRAG calibration/README.md

```diff
@@ -0,0 +1,75 @@
+# Derivative Removal by Adiabatic Gate (DRAG) calibration
+
+_Author: Kevin Villegas Rosales_
+
+_Important note: The code in this folder is the exact code that we used
+to run the experiment, so it is tailored for the specific software environment
+and set up. Thus, this is code is for inspirational purposes._
+
+## The goal
+
+The goal of this experiment is to calibrate the DRAG coefficient
+to increase the single qubit gate fidelities as measured with 
+randomized benchmarking (RB).
+
+## The device
+
+The chip contains 5 2D-transoms, where some of them have nearest-neighbor connectivity, 
+with flux tunable lines. For this experiment, we focus only in one transmon with a drive
```

Comment:
This has actually been performed on a flux tunable transmon so I would move it to the flux tunable transmon folder

Reply:
There is no flux element only that I have added a bias in the controllers that is why I put in the fixed transmon one, I'm ambivalent about this